### DISCUSSION TITLE
Changing width and height of element

### DISCUSSION THREAD

**Discussion by makleo2002** [clientIO/joint#2660] [Q&A]
### Introduction

I have a properties panel where I can change the length and height of an element. I also have a Halo. When I change the length and height through the properties panel, my halo breaks (I resize the shape through it, but only the nodes and the boundary tool move). How can I fix this?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
You can use `element.resize()` as mentioned in our [element tutorial](https://resources.jointjs.com/tutorial/elements).
Also check out [special attributes tutorial](https://resources.jointjs.com/tutorial/special-attributes), so you understand how to use `calc()` to define a relative size of an SVG Element to the model's size.

If you want us to help you better, always remember to provide us with a code.

**Reply by makleo2002** [reply]
why is this code not working?how do I access the attributes of the elements correctly?
`propWidth.value = currentElement.attr('body/width');
    propHeight.value = currentElement.attr('body/height');`

**Reply by kumilingus** [reply]
And what is it supposed to do? What is `propWidth`?

Here's how to ask a good technical question: https://github.com/clientIO/joint/wiki/Asking-Technical-Questions
